lib/GRNOC/NetSage/Deidentifier/NetflowImporter.pm

```diff
@@ -380,6 +380,12 @@ my $path = $params{'path'};
 
             my $sum_bytes = $ibyt + $obyt;
             my $sum_packets = $ipkt + $opkt;
+            my $proto = '';
+            if($pr =~ /d+/){
+                $proto = getprotobynumber( $pr );
+            }else{
+                $proto = getprotobyname( lc($pr) );
+            }
```

Comment:
formatting/spacing , but otherwise LGTM.

Reply:
I updated the spacing to match the other if statements in the file. Thanks.

Reply:
Also found an issue with getprotobyname returning the protocol number, so dropped that call since it wasn't really necessary anyways